### PR TITLE
Increases specificity of search related selectors to work with recent wagtail updates

### PR DESCRIPTION
**Pull request by SaptakS**
In the latest wagtail updates, wagtail has increased the specficity of some of their input forms, which causes the wagtail search to not get the proper styling. I have added input to increase the specificity of wagtail autocomplete styles which fixes the issues.

Before this PR, in wagtail 2.11.5:
image

### CODE REVIEW

**Review by harrislapiroff — DISMISSED** [review]
Looks good!

**Review by harrislapiroff — APPROVED** [review]
Looks great, let's do it!

File: client/src/components/AutocompleteInput/_search.scss [hunk 1]
```diff
@@ -7,11 +7,11 @@
     }
 }
 
-.c-wagtailautocomplete__search-input-container {
+input.c-wagtailautocomplete__search-input-container {
```
Comment (ARogovskyy) [inline client/src/components/AutocompleteInput/_search.scss:10]: But the container (and the icon) aren't inputs, so this style won't apply.

File: client/src/components/AutocompleteInput/_search.scss [hunk 2]
```diff
@@ -7,11 +7,11 @@
...
     position: relative;
 }
 
-.c-wagtailautocomplete__search-icon {
+input.c-wagtailautocomplete__search-icon {
```
Comment (ARogovskyy) [inline client/src/components/AutocompleteInput/_search.scss:14]: (see above)

Comment (oladhari) [reply client/src/components/AutocompleteInput/_search.scss:14]: thank you for checking it

**Comment by SaptakS** [comment]
@ARogovskyy @oladhari https://github.com/wagtail/wagtail-autocomplete/pull/89/files should fix the isssue. Feel free to review, and let me know. Sorry for the stupid mistake I made.

**Pull request merged**